Documentation/ceph-upgrade.md

```diff
@@ -221,6 +221,16 @@ Replace the namespace names in the new resources:
 sed "s/ROOK_SYSTEM_NAMESPACE/$ROOK_SYSTEM_NAMESPACE/g" upgrade-from-v1.0-create.yaml > upgrade-from-v1.0-create.yaml.tmp
 sed "s/ROOK_NAMESPACE/$ROOK_NAMESPACE/g" upgrade-from-v1.0-create.yaml.tmp > upgrade-from-v1.0-create.yaml
 rm -f upgrade-from-v1.0-create.yaml.tmp
+sed "s/ROOK_NAMESPACE/$ROOK_NAMESPACE/g" upgrade-from-v1.0-apply.yaml
+```
+
+If you have a v1.0 cluster running with CSI drivers enabled, delete the rbac rules created for CSI
+
+```sh
+kubectl delete -f https://raw.githubusercontent.com/rook/rook/release-1.0/cluster/examples/kubernetes/ceph/csi/rbac/cephfs/csi-nodeplugin-rbac.yaml
```

Comment:
Instead of deleting these via files stored on GitHub, we should instruct the users to delete the RBAC permissions via `kubectl --namespace $NAMESPACE delete <resource> <name>` commands so the commands work for their environment.

Reply:
done